packages/editor/src/client/locales/en.json

```diff
@@ -643,7 +644,8 @@
         "token": "Token",
         "Unknown": "Unknown",
         "URL": "URL",
-        "validateToken": "Please confirm token is correct before saving"
+        "validateToken": "Please confirm token is correct before saving",
+        "leaveEmpty": "Leave empty if you don't want to change it"
       }
```

Comment:
Shouldn't those also come into fr.json and de.json ?

Reply:
@camilleory translations happen with a tool once the new translation string have been merge to `master`